apps/ledger-live-desktop/src/renderer/screens/settings/sections/Experimental/PostOnboardingHubTester.tsx

```diff
@@ -14,38 +14,72 @@ const PostOnboardingHubTester = () => {
   const handleInitPostOnboarding = useStartPostOnboardingCallback();
 
   return (
-    <SettingsSectionRow
-      title={t("settings.experimental.features.testPostonboarding.title")}
-      desc={t("settings.experimental.features.testPostonboarding.description")}
-    >
-      <Flex flexDirection={"row"} columnGap={3}>
-        <Button
-          data-test-id="postonboarding-tester-button"
-          onClick={() =>
-            handleInitPostOnboarding({
-              deviceModelId: DeviceModelId.stax,
-              mock: true,
-              fallbackIfNoAction: () => history.push("/"),
-            })
-          }
-          primary
-        >
-          {t("postOnboardingDebugger.buttonTitle")} (mock)
-        </Button>
-        <Button
-          onClick={() =>
-            handleInitPostOnboarding({
-              deviceModelId: DeviceModelId.stax,
-              mock: false,
-              fallbackIfNoAction: () => history.push("/"),
-            })
-          }
-          primary
-        >
-          {t("postOnboardingDebugger.buttonTitle")}
-        </Button>
-      </Flex>
-    </SettingsSectionRow>
+    <>
+      <SettingsSectionRow
+        title={t("settings.experimental.features.testStaxPostonboarding.title")}
+        desc={t("settings.experimental.features.testStaxPostonboarding.description")}
+      >
+        <Flex flexDirection={"row"} columnGap={3}>
+          <Button
+            data-test-id="stax-postonboarding-tester-button"
+            onClick={() =>
+              handleInitPostOnboarding({
+                deviceModelId: DeviceModelId.stax,
+                mock: true,
+                fallbackIfNoAction: () => history.push("/"),
+              })
+            }
```

Comment:
A bit of a quibble, but we can simplify the code a bit by imagining that one day we'll add a 3rd device.

```suggestion
   <>
  {[DeviceModelId.europa, DeviceModelId.stax].map((deviceId) => (
    <SettingsSectionRow
      key={deviceId}
      title={t(`settings.experimental.features.test${deviceId}Postonboarding.title`)}
      desc={t(`settings.experimental.features.test${deviceId}Postonboarding.description`)}
    >
      <Flex flexDirection={"row"} columnGap={3}>
        <Button
          data-test-id={`${deviceId.toLowerCase()}-postonboarding-tester-button`}
          onClick={() =>
            handleInitPostOnboarding({
              deviceModelId: deviceId,
              mock: true,
              fallbackIfNoAction: () => history.push("/"),
            })
          }
          primary
        >
          {t("postOnboardingDebugger.buttonTitle")} (mock)
        </Button>
        <Button
          onClick={() =>
            handleInitPostOnboarding({
              deviceModelId: deviceId,
              mock: false,
              fallbackIfNoAction: () => history.push("/"),
            })
          }
          primary
        >
          {t("postOnboardingDebugger.buttonTitle")}
        </Button>
      </Flex>
    </SettingsSectionRow>
  ))}
</>
```